aviary/docs/examples/modified_aircraft.csv

```diff
@@ -89,7 +89,7 @@ aircraft:horizontal_tail:vertical_tail_fraction,0.0,unitless
 aircraft:horizontal_tail:wetted_area_scaler,1.0,unitless
 aircraft:horizontal_tail:wetted_area,592.65,ft**2
 aircraft:hydraulics:mass_scaler,1.0,unitless
-aircraft:hydraulics:system_pressure,3000,psi
+aircraft:hydraulics:system_pressure,3000,lbf/ft**2
```

Comment:
I'm going to need significantly more convincing that these systems were actually designed to work at 21 PSI.

3,000 lbf/ft^2 = 21 PSI

Reply:
![image](https://github.com/user-attachments/assets/3b14204e-d16d-4818-b244-f39b41bb7c57)
See page 38 of the FLOPS hydraulic systems weight calculations: https://ntrs.nasa.gov/api/citations/20170005851/downloads/20170005851.pdf
They are clearly showing that sys_press = 3000 psi

Reply:
Yeah, that is pretty definitive. I am going to cancel this PR and figure out what is going on locally.